stdlib/ostruct.rb

```diff
@@ -60,10 +66,52 @@ def to_n
   def hash
     @table.hash
   end
+  
+  def table
+    @table
+  end
+  
+  def delete_field(name)
+    sym = name.to_sym
+    begin
+      singleton_class.__send__(:remove_method, sym, "#{sym}=")
+    rescue NameError
+    end
+    @table.delete sym
+  end
+  
+  def new_ostruct_member(name)
```

Comment:
It doesn't look to me like `#table` is a method at all, since it's returning `nil`, it looks like to me it just goes through the normal `method_missing` route.

Yeah, I'm fine with using `@table`.